app/src/main/kotlin/com/aliucord/manager/MainActivity.kt

```diff
@@ -47,7 +46,9 @@ class MainActivity : ComponentActivity() {
                     if (!navigator.pop()) finish()
                 }
 
-                ManageStorageDialog()
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+                    ManageStorageDialog()
+                }
```

Comment:
Just make this a one line if statement, guh

Reply:
INSANE

Reply:
One line one line